hack/make-rules/autok3s.sh

```diff
@@ -275,20 +150,17 @@ function unit() {
 
   local os="${OS:-$(go env GOOS)}"
   local arch="${ARCH:-$(go env GOARCH)}"
+  local race_tag="-race"
   if [[ "${arch}" == "arm" ]]; then
     # NB(thxCode): race detector doesn't support `arm` arch, ref to:
     # - https://golang.org/doc/articles/race_detector.html#Supported_Systems
+    race_tag=""
+  fi
     GOOS=${os} GOARCH=${arch} CGO_ENABLED=1 go test \
```

Comment:
Do we also need to DISABLE CGO here for the test?

Reply:
The `race` tags for go test requires CGO enable.